cmake/Modules/ObsDefaults_Windows.cmake

```diff
@@ -0,0 +1,123 @@
+cmake_minimum_required(VERSION 3.20)
```

Comment:
The root CMakeLists.txt changed the minimum to 3.16...3.21.  Should this also be changed?

Reply:
I think on Windows we set it to 3.20 as people need to install it manually anyway and we wanted to get a very recent version and the `3.16...` just exists to make cmake exit early on older Linux'.

I'm hesitant to remove it entirely from the per-OS settings as especially on macOS some very recent features are required (e.g. for Xcode compatibility) so 3.20 is a must there.

Given that it's a "MINIMUM REQUIRED" 3.20 should cover newer versions anyway.

Reply:
Seems fine then.